models/src/main/scala/coop/rchain/models/rholang/sort/ScoreTree.scala

```diff
@@ -26,7 +26,7 @@ trait ScoreTree {
   case class Node[T](children: Seq[Tree[T]]) extends Tree[T]
 
   sealed trait TaggedAtom
-  case class IntAtom(i: Int)          extends TaggedAtom
+  case class IntAtom(i: Long)         extends TaggedAtom
```

Comment:
Before anybody asks - I've heard we're heading to arbitrary-length integers in the long term, so it makes little sense to rename to `LongFoo`-s just to go back to `[Big]Int[eger]Foo`-s. I've only done the rename for `evalToInt -> evalToLong`, where it actually matters what concrete integer type is returned.